lib/plant.py

```diff
@@ -1,4 +1,23 @@
+import time
+import board
+import busio
+from adafruit_seesaw.seesaw import Seesaw
+
 class Plant():
 
     def soil_moisture(self):
-        return 350
+
+        i2c_bus = busio.I2C(board.SCL, board.SDA)
+
+        ss = Seesaw(i2c_bus, addr=0x36)
+
+        while True:
```

Comment:
Will this `while True` argument be extracted to the controller when it's created? Otherwise when will this loop break and return something to interact with the Light class?

Reply:
Yes, it should be extracted for the reasons stated